docker.d/pre-stop.sh

```diff
@@ -30,13 +30,11 @@ POLL_INTERVAL=5
 
 started=$(date +%s)
 
-echo "Sending SIGUSR1 signal to process id $SCRIPTWORKER_PID"
 
-kill -s USR1 $SCRIPTWORKER_PID
 
 while true; do
-    echo "checking status..."
-    kill -0 $SCRIPTWORKER_PID
+    echo "Sending SIGUSR1 signal to process id $SCRIPTWORKER_PID"
```

Comment:
nit:
```suggestion
    echo "Sending SIGUSR1 to process id $SCRIPTWORKER_PID"
```

Also, I'm wondering how useful it is to be repeating that line every 5 seconds?